webservice/template/admin/index.tmpl

```diff
@@ -53,7 +53,7 @@
                         <span class="badge {{if DEADLINEDUE .Assignment.Deadline}}badge-dark{{ else}} badge-secondary{{end}}">{{.CourseCode}}</span>
                     </h5>
                     <p{{if DEADLINEDUE .Assignment.Deadline}} class="text-danger"{{end}}>
-                        <strong>Deadline: </strong> {{PRETTYTIME .Assignment.Deadline}}</p>
+                        <strong>Deadline: </strong> {{PRETTYTIME .Assignment.Deadline}} <i>(Norwegian time!)</i></p> <!-- TODO time-norwegian -->
```

Comment:
Remove "(Norwegian time!)"

Reply:
Christopher asked for this specific, so nah